doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
+
+## What's included in a free trial?
+
+In Materialize, [cluster replicas](/overview/key-concepts/#cluster-replicas) are the physical resources for doing computational work. Cluster replicas come in various sizes, and each size corresponds to a specific number of [compute credits](https://materialize.com/pricing/).
+
+During your free trial, you get up to 4 compute credits per region at any given time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
```

Comment:
```suggestion
During your free trial, your credit consumption rate in a given region cannot exceed 4 credits per hour at any given time. This limit should accommodate most trial scenarios. If you need additional resources during your trial, please contact sales@materialize.com.
```

Reply:
Done. This is much more precise, but I do worry that it's nonetheless complex to digest. think I might add an example.